public/index.php

```diff
@@ -1,5 +1,6 @@
 <?php
 
+$name = 'Oxygen';
```

Comment:
this variable has no place in this file

Reply:
Where can I put this variable? In the HomeController or layout.html.twig ?